File: LatB/Java/Ram.java
public class Ram{
    // attribute class Ram
    private String kapasitas;
    private int harga;
    // Constructor
    public Ram(){}
    public Ram(String kapasitas, int harga) {
        this.kapasitas = kapasitas;
        this.harga = harga;
    }
    // Get dan Set
    public String getKapasitas() {
        return kapasitas;
    }
    public void setKapasitas(String kapasitas) {
        this.kapasitas = kapasitas;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
    // Fungsi untuk menampilkan isi dari attribute class Ram
    public void printRam(){
        System.out.println("Kapasitas Ram: " + this.kapasitas);
        System.out.println("Harga Ram: " + this.harga);
    }   
}

File: LatB/Php/Ram.php
<?php
class Ram{
    # Attribute untuk class Ram
    private $kapasitas;
    private $harga;
    # Constructor dengan parameter dari attribute private
    function __construct($kapasitas, $harga)
    {
        $this->kapasitas = $kapasitas;
        $this->harga = $harga;
    }
    # Get dan Set semua attribute private
    function setKapasitas($kapasitas){
        $this->kapasitas = $kapasitas;
    }
    function getKapasitas(){
        return $this->kapasitas;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    # Fungsi untuk menampilkan isi dari attribute private
    function printRam(){
        echo "RAM Capacity: " .$this->kapasitas."<br>";
        echo "RAM Price: " .$this->harga."<br>";
    }
    # Destructor
    function __destruct()
    {
        
    }
}
?>